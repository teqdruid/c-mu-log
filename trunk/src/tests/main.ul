/* test to find the no. solutions one gets under given constraints*/

wall(3,4);
wall(4,8);
wall(6,8);
/*wall($X,$Y) {AND:
	$X>=10;
	$X<=15;	
	$Y<=8;
	$Y>=2;
	$X=aabfajds;
	}*/
wall(){}

move("UP");
move("UP");
move("UP");

main ()
{
    wall(3,?);
//	!wall(4,5);	
    
     
{OR:
	@print("Wall: ", $x, ",", $y);
	wall($x, $y);
}

}

		
