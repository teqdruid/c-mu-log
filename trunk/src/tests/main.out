Hello

 ^^^  Solution  ^^^ 

Hello

 ^^^  Solution  ^^^ 

Hello

 ^^^  Solution  ^^^ 

Hello

 ^^^  Solution  ^^^ 

Hello

 ^^^  Solution  ^^^ 

Hello

 ^^^  Solution  ^^^ 

Hello

 ^^^  Solution  ^^^ 

Hello

 ^^^  Solution  ^^^ 

Hello

 ^^^  Solution  ^^^ 

No more solutions
